extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -121,6 +121,7 @@ function render_player( $player_data, $attributes ) {
 
 	$player_classes_name = trim( "{$secondary_colors['class']} {$background_colors['class']}" );
 	$player_inline_style = trim( "{$secondary_colors['style']} ${background_colors['style']}" );
+	$player_inline_style .= get_css_vars( $attributes );
```

Comment:
Does phpcs warn about unaligned equal signs?

Reply:
It didn't at least to me. Going to align.